ui/app/AppLayouts/Profile/views/profile/MyProfileSettingsView.qml

```diff
@@ -182,6 +198,7 @@ ColumnLayout {
         StatusTabButton {
             width: implicitWidth
             text: qsTr("Collectibles")
+            enabled: false // TODO: implement collectibles nim part
```

Comment:
Do we have an issue to track this?

Reply:
And, can we keep the "Collectibles" tab with the dummy backend for now, until we do?

Reply:
Yes, going to ship it in this PR: https://github.com/status-im/status-desktop/pull/12594 (waiting for https://github.com/status-im/status-desktop/issues/12578)

And there is an issue with dummy backend currently ,because when wallet is loading it shows 15 placeholders which can pollute showcase model

Reply:
Yeah I see, good enough and great to hear it's in progress!